src/Control/Effect/Catch/Internal.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE DeriveFunctor, ExistentialQuantification, StandaloneDeriving #-}
+module Control.Effect.Catch.Internal
+( Catch(..)
+) where
+
+import Control.Effect.Class
+
+-- | 'Catch' effects can be used alongside 'Control.Effect.Throw.Throw' to provide recoverable exceptions.
+--
+-- @since 1.0.0.0
```

Comment:
I double-checked that haddock propagates these docs along to `Control.Effect.Catch`.